cart.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en"><head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles/csrt.css">
```

Comment:
I guess you have a typo in the file name, cart not csrt, right?

[This vscode extention ](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) will help you catch typos.